mat-process-api/V1/Domain/ProcessData.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace mat_process_api.V1.Domain
+{
+    [BsonIgnoreExtraElements]
+    public class MatProcessData
+    {
+        [JsonProperty("_id")]
+        [BsonRepresentation(BsonType.ObjectId)]
+        public string Id { get; set; }
+        [JsonProperty("processType")]
+        [BsonElement("processType")]
+        public string ProcessType { get; set; }
+        [JsonProperty("dateCreated")]
+        [BsonElement("dateCreated")]
+        public DateTime DateCreated { get; set; }
+        [JsonProperty("dateLastModified")]
+        [BsonElement("dateLastModified")]
+        public DateTime DateLastModified { get; set; }
+        [JsonProperty("dateCompleted")]
+        [BsonElement("dateCompleted")]
+        public DateTime DateCompleted { get; set; }
+        [JsonProperty("dataSchemaVersion")]
+        [BsonElement("dataSchemaVersion")]
+        public int DataSchemaVersion { get; set; }
```

Comment:
Suggest we call this `ProcessDataSchemaVersion`, as it refers directly to the `ProcessData`. I can imagine a case where we want to have difference schemas for the different data blobs.

Reply:
changed